docs/source/configuration/traffic-shaping.mdx

```diff
@@ -52,7 +52,7 @@ This rate limiting applies to all requests, there is no filtering per IP or othe
 
 ### Timeout
 
-The Apollo Router applies a default limit of 30 seconds to receive the entire client request. That limit is configurable:
+The Apollo Router applies a default limit of 30 seconds to receive the entire client request. The default limit of 30 seconds also applies for request between router to subgraphs. That limit is configurable:
```

Comment:
```suggestion
The Apollo Router applies a default limit of 30 seconds for all requests. This timeout applies to requests the client makes to the router and between the router and subgraphs. That limit is configurable:
```
Just rewording to emphasize the 30s timeout and then describe it further.

Just want to double-check: does this timeout apply to
- 1) requests the router make to subgraphs
- 2) requests the subgraphs make to the router in case of HTTP subscriptions

If it's both, then this works well. If it's only 1, then we should specify that.

Reply:
It also applies to subscription request but as subscription http request is a long lived connection the timeout is enabled until we received first response headers but once we received these response headers the connection can live for longer than 30secs

Reply:
@Meschreiber It could also be interesting to solve [this issue](https://github.com/apollographql/router/issues/3441) too, I think it's related. I added a comment in the issue to clarify

Reply:
Added a note about subscriptions and tweaked the suggestion above

Reply:
Thanks @BrynCooke I'll wait on the feedback from @Meschreiber to also solve [this issue](https://github.com/apollographql/router/issues/3441) and we're good to merge

Reply:
Ah, thanks for flagging @bnjjj ! Stacked this PR rather than adding more comments: https://github.com/apollographql/router/pull/3472/files

Hopefully I've understood the configuration options correctly.

Reply:
@lleadbet , thanks for calling that out, will add!